pubsub/api/src/create_unwrapped_push_subscription.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+/**
+ * Copyright 2025 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * For instructions on how to run the full sample:
+ *
+ * @see https://github.com/GoogleCloudPlatform/php-docs-samples/blob/main/pubsub/api/README.md
+ */
+
+namespace Google\Cloud\Samples\PubSub;
+
+# [START pubsub_create_unwrapped_push_subscription]
+use Google\Cloud\PubSub\PubSubClient;
+
+/**
+ * Create unwrappped push subscription.
+ *
+ * @param string $projectId  The Google project ID.
+ * @param string $topicName  The Pub/Sub topic name.
+ * @param string $subscriptionId  The ID of the subscription.
+ */
+function create_unwrapped_push_subscription(
+    string $projectId,
+    string $topicName,
+    string $subscriptionId
+): void {
+
+    $pubsub = new PubSubClient([
+        'projectId' => $projectId,
+    ]);
+    $pubsub->subscribe($subscriptionId, $topicName, [
+        'pushConfig' => [
+            'no_wrapper' => [
+                'write_metadata' => true
+            ]
```

Comment:
so, it's a bit confusing, but we accept both snake and camel-case for fields which are serialized into protobuf messages from handwritten client libraries

I am a bit torn because on one hand, we use camelCase across handwritten and GAPIC libraries for all options. On the other hand, protobuf uses snake_case for everything. However, I think it's important that we stay consistent, and for now I prefer camelCase to snake_case.

```suggestion
            'noWrapper' => [
                'writeMetadata' => true
            ]
```

Reply:
Sounds good. Do you want this change to be applied for other tests as well?